scripts/LedSignMock.py

```diff
@@ -0,0 +1,35 @@
+import curses
+import argparse
+
+parser = argparse.ArgumentParser(description="Prints mock output.")
+parser.add_argument('--mock-output', help = "Displays Mock Output", )
+args = parser.parse_args()
+
+menu = ['   ','SCE LED Sign Visualization ', ' ','Most Recent Health Check: ', 'Most Recent Sign Update: ',' ',
+'Current Sign Text:', 'Current Brightness:', 'Current Scroll Speed:', 'Current Text Color: ', 'Current Background:',
+'Current Border Color: ' ]
+
+class LedSignMock(data):
+    def print_menu(self, stdscr, selected_row_idx):
+        data = "info here"
+        stdscr.clear()
+        stdscr.border(0)
+        height, width = stdscr.getmaxyx()
+        for idx, row in enumerate(menu):
+            x = width//3 - len(menu)
+            y = height//3 - len(menu)//3 + idx
+            stdscr.addstr(y, x, row)
+        stdscr.refresh()
+
+    def creates_display(self, stdscr):
+        curses.curs_set(0)
+        current_row = 0
+        print_menu(stdscr, current_row)
+        checkKey = stdscr.getch()
+        while checkKey != 27:
```

Comment:
I think Evan was correct about creating a constant for the number 27. This way anyone who goes through this code in the future will know exactly what this is doing.